### DISCUSSION TITLE
Filters as nav

### DISCUSSION THREAD

**Discussion by NoWorries** [shishkin/astro-pagefind#50] [Q&A] [answered]
Is there a way to use pagefind in combination with the filters as a way to navigate, as well as refine results?

**Comment by shishkin** [comment]
Hi @NoWorries, I'm not sure what you mean. Do you have an example of a website doing it?

This project is an Astro wrapper for [pagefind](https://pagefind.app/) library. If you can do it with bare pagefind, there could also be a way to do it with pagefind in Astro.

**Comment by NoWorries** [comment]
Hey yeah maybe better question for pagefind. I was curious if you could use
the filters as a faceted search. So you'd start with all the results and be
able to filter, sort, and search from the entire set at the start.

**Reply by shishkin** [reply, marked as answer]
I just took a look at pagefind docs and it seem to provide [filters in the default UI](https://pagefind.app/docs/default-ui-filtering/). Does it do what you need?

**Comment by sreejithkalarikkal** [comment]
Hi I have an SSR site i included the code for SSR site search as in the document ,but it not working .Could you please help on this

**Reply by shishkin** [reply]
SSR is not supported as doesn't make sense for pre-built search index.
